shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/messaging/MessageReaderAdapterTest.java

```diff
@@ -231,11 +247,13 @@ public void metadataOnlySyncEventForPartitionWithNullLocations() {
     MetaStoreEvent actual = messageReaderAdapter.read().get();
 
     assertMetaStoreEvent(expected, actual);
+    verify(messageReader, timeout(1)).delete(RECEIPT_HANDLE);
```

Comment:
Trying to understand why we need to check for timeout(1) here ?

Reply:
Good spot, this should be `times(1)`, will change